crosswalks/spase-v2.6.1.csv

```diff
@@ -0,0 +1,74 @@
+Property,SPASE
+codeRepository,RepositoryID
+programmingLanguage,codeLanguage
+review,
+runtimePlatform,
+targetProduct,
+applicationCategory,
+applicationSubCategory,
+downloadUrl,AccessURL/URL
+fileSize,
+installUrl,ExecutionEnvironment/Installer/URL
+memoryRequirements,ExecutionEnvironment/Memory
+operatingSystem,OperatingSystem
+permissions,
+processorRequirements,
+releaseNotes,
+softwareHelp,
+softwareRequirements,
+softwareVersion,SoftwareVersion
+storageRequirements,ExecutionEnvironment/Storage
+supportingData,
+author,
+citation,
+contributor,
+copyrightHolder,
+copyrightYear,
+dateCreated,
+dateModified,
+datePublished,ResourceHeader/ReleaseDate
+editor,
+encoding,AccessInformation/Encoding
+fileFormat,AccessInformation/Format
+funder,ResourceHeader/Funding
+keywords,
+license,
+producer,
+provider,
+publisher,
+sponsor,
+version,
+isAccessibleForFree,
+isPartOf,
+hasPart,
+position,
+description,ResourceHeader/Description
+identifier,ResourceID
+name,ResourceHeader/ResourceName
+sameAs,
+url,
+relatedLink,
+givenName,
+familyName,
+email,
+affiliation,
+identifier,ResourceHeader/DOI
+name,
+address,
+reviewAspect,
+reviewBody,
+endDate,
+roleName,
+startDate,
+softwareSuggestions,
+maintainer,
```

Comment:
this should be ResourceHeader/Contact (http://www.spase-group.org/data/schema/Contact)